R/recode2NA.R

```diff
@@ -40,15 +40,18 @@ recode2NA <- function(GADSdat, recodeVars = namesGADS(GADSdat), value = "") {
 recode2NA.GADSdat <- function(GADSdat, recodeVars = namesGADS(GADSdat), value = "") {
   check_GADSdat(GADSdat)
   if(!is.character(recodeVars) || length(recodeVars) < 1) stop("'recodeVars' needs to be character vector of at least length 1.")

```

Comment:
Nit: I would prefer a two-line statement here for the sake of readibility:

```r
if (!is.character(recodeVars) || length(recodeVars) < 1) {
   stop("'recodeVars' needs to be character vector of at least length 1.")
}


```

Reply:
Fixed!